src/main.cpp

```diff
@@ -15,20 +16,23 @@ int main(int argc, char *argv[]) {
 	if (argv[argc - 1] == std::string("-h") ||
 			argv[argc - 1] == std::string("-help")) {
 		Frontend::PrintHelper();
-		return 0;
+		return EX_OK;
 	}
 
 	for (int i = 1; i < argc; ++i) {
 		if (file_included(argv[i])) {
 			filename = argv[i];
+		} else if (!file_included(argv[i])) {
```

Comment:
Når du laver koden sådan der, kan du aldrig skrive `chemilang -o out.crn example.chem` uden at den prøver at parse `-o` som en fil.

Reply:
I know men kan ikke lige se en anden måde at håndter det på? Der skal jo være en eller anden form for struktur.

Reply:
I det mindste burde det prøve at tjekke om det er et argument *før* den tjekker om det er en fil

Reply:
jeg prøver at kigge efter et library til at gøre det her..

Reply:
Vi har ikke brug for et library, du skal bare flytte den her else if ned i bunden

Reply:
Altså den er jo lavet sådan at man skal skrive ./chemlang [filename] [Option]

Reply:
Vent hvad, er det det vi har skrevet i help? Det skal fixes, for sådan er der ikke nogle unix programmer der virker. 

In any case, det vil heller virke, fordi den stadig vil prøve at parse `-o` som et filnavn.

Reply:
Yeah. Jeg lavede vare help function på baggrund af det der allerede var i main